css/faq_home.css

```diff
@@ -0,0 +1,78 @@
+#faq-menu {
+    padding: 30px 15px 0;
+}
+
+#faq-menu .tile {
+    display: flex;
+    background-color: rgba(226, 222, 224, 0.1);
+    color: white;
+    padding: 45px 0;
+    transition: transform 0.25s ease-in-out, filter 0.25s ease-in-out;
+}
+
+#faq-menu .tile:hover {
+    transform: scale(1.1);
+    z-index: 3;
+    filter: brightness(1.3);
+}
+
+#faq-menu .tile::before {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    content: '';
+    /* fallback color is combination of #CD201F and #AA1D1D */
+    background: #BC1F1E url(/img/bg_rectangle_tile.svg) no-repeat center;
```

Comment:
A relative URL would make more sense (in CSS, those are relative to the file anyway, right?), if we ever introduced some "prefix" like Jekyll allows for in the config, this would break.